package.json

```diff
@@ -1,91 +1,42 @@
 {
-  "name": "graphql-hooks",
-  "version": "3.2.2",
-  "description": "Graphql Hooks",
-  "main": "lib/graphql-hooks.js",
-  "module": "es/graphql-hooks.js",
-  "unpkg": "dist/graphql-hooks.min.js",
+  "name": "root",
+  "private": true,
   "scripts": {
     "test": "jest",
-    "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
-    "eslint": "eslint src test",
-    "prettier": "prettier src/**/*.js test/**/*.js --write",
-    "build": "rollup -c",
-    "prepublishOnly": "npm run build"
-  },
-  "files": [
-    "dist",
-    "es",
-    "lib"
-  ],
-  "keywords": [
-    "graphql",
-    "hooks",
-    "graphql-hooks",
-    "gql",
-    "gql-hooks",
-    "react-graphql-hooks",
-    "react",
-    "apollo"
-  ],
-  "author": "Brian Mullan <bmullan91@gmail.com>",
-  "license": "Apache-2.0",
-  "peerDependencies": {
-    "react": "^16.8.3"
+    "postinstall": "lerna bootstrap --no-ci",
+    "coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "eslint": "eslint packages/*/{src,test}/**/*.js",
```

Comment:
Both `graphql-hooks-ssr` and `graphql-hooks-memcache` don't have a `src` folder - also we should catch top-level .js files.

I _think_ this catches them all: `eslint *.js packages/*/*.js packages/*/{src,test}/**/*.js`